src/hooks/useMfe.js

```diff
@@ -4,11 +4,8 @@ import { getMfeById } from 'state/mfe/selectors';
 
 
 // try to find the element by id
-const isLoaded = (id) => {
-  const script = document.getElementById(`script-${id}`);
-  const style = document.getElementById(`style-${id}`);
-  return script && style;
-};
+const isJSLoaded = id => document.getElementById(`script-${id}`);
+const isCSSLoaded = id => document.getElementById(`style-${id}`);
```

Comment:
the CSS asset can be optional. Some MFEs don't export an external CSS because they inject it directly into the shadow dom. For this reason, we need to decouple the control regarding the check for the style loading. This prevents a bug of infinite loop when an MFE without external CSS is loaded.